### ISSUE TITLE
Support to Microsoft SQL Server OR ignore alias "time"

### ISSUE THREAD

**Issue by edum18** [Vonage/Grafana_Status_panel#123]
The only way to use a Microsoft SQL Server data source on this plugin is formatting query as "Time Series" which implies that on the sql query there must be a column with alias "time". So if we want multiple metrics, there will be multiple queries and the same "time" column on each one. But the panel doesn't allow same alias which will never work.
So is it planned to support this data source type? A possible fix that I was thinking about could be to "ignore" the alias called "time"...
Thank you.

**Comment by syedsfayaz** [comment]
@edum18 I am seeing the same issue. I am using mysql database. Is there a workaround for this issue?. Is there any other panel in grafana where we can display multiple columns.

**Comment by edum18** [comment]
@syedsfayaz 
My guess is to write some HTML and javascript and show it on a text panel. With this you have total freedom of what to show there. But to get query results you would need to create variables and show them on the panel. I didn't test it yet though.

**Comment by syedsfayaz** [comment]
@ thank you. I was trying to test it with influx db which is supported by this panel.

**Comment by edum18** [comment]
> @ thank you. I was trying to test it with influx db which is supported by this panel.

@syedsfayaz  In case of you still interested, I managed to get it working with SQL server by deleting the line "this.duplicates = true;" of the plugin source so that it never detects duplicates

**Comment by syedsfayaz** [comment]
@edum18 Thanks for your response. I tried commenting  out that line from the src file "status_ctrl.js " but that did not help me. I am seeing the same "Error: There are multiple metrics with the same alias. Please give each metric a unique name." Here is my code.  

 if (res.length > 1) {
                        **/**     this.duplicates = true;**/**
                        }



I also see other line with (this.duplicates ),Did you just delete one line  or made any other changes as part of this fix.

**Comment by edum18** [comment]
> @edum18 Thanks, I was able to get it working. I had to make changes in both src and dist directory.

doing that works but doesnt work anymore if you update the plugin for some newer version. So if you do, change it again. Btw you just need to change in the dist directory because that's what it is running

**Comment by syedsfayaz** [comment]
@edum18 Agree. I have one more scenario where I am having issue displaying String data. One of my column in the table has string data like environment name. This panel shows only numeric data like build number, time but does not show string values. Any idea if you have come across this specific scenario?

**Comment by edum18** [comment]
> @edum18 Agree. I have one more scenario where I am having issue displaying String data. One of my column in the table has string data like environment name. This panel shows only numeric data like build number, time but does not show string values. Any idea if you have come across this specific scenario?

nice catch, I can't make it work too. the plugin probably need some more changes

**Comment by yesoreyeram** [comment]
https://github.com/yesoreyeram/yesoreyeram-boomsummary-panel  may help somehow. (Still in poc/Dev stage)

**Comment by syedsfayaz** [comment]
@yesoreyeram  Will check it out. Thankyou

**Comment by syedsfayaz** [comment]
@yesoreyeram 

Is there a feasibility to change the Dimension of this panel?. Like I want to show it as below. 

In my case I have multiple environments from where the package flows. 

`Env 1   -- top page
Dev --> QA --> UAT -->  Prod   -- After drill down. `

Here I want to show the build number and some other results from jenkins job. Is there a way I can show multiple status. I will be pulling results from a jenkins build and if the build is still running I want to show a flashing orange. Can I make this Panel work like that?
Attached sample for idea. 

![image](https://user-images.githubusercontent.com/26099918/56148410-13c94480-5f5f-11e9-99d1-63842809ffda.png)
